migrating/cluster-wide/ossm-migrating-cluster-wide-assembly.adoc

```diff
@@ -15,4 +15,21 @@ You must complete the premigration checklists before you start migrating your de
 
 include::modules/ossm-control-plane-configuration-migration-requirements.adoc[leveloffset=+1]
 
-include::modules/ossm-cluster-wide-migration-methods.adoc[leveloffset=+1]
\ No newline at end of file
+include::modules/ossm-cluster-wide-migration-methods.adoc[leveloffset=+1]
+
+include::modules/ossm-migrating-a-cluster-wide-deployment-using-the-istio-injection-label.adoc[leveloffset=+1]
+
+include::modules/ossm-migrating-workloads-using-the-istio-injection-label.adoc[leveloffset=+2]
+
+.Next steps
+
+If you are using gateways, you must migrate them before you complete the migration process for your deployment and workloads.
+
+* Migrate gateways
+//<insert xref to gateway migration content>
+//xrefs across Migration Guides will be handled by OSSM-8852
+migrating/migrating-gateways/ossm-migrating-gateways-assembly.adoc
+
+If you are not using gateways, and have verified your cluster-wide migration, you can proceed to complete the migration and remove {SMProduct} 2 resources.
+
```

Comment:
We have to include this warning as it's very important:
`Before creating a default revision tag and re-label namespaces, it's necessary to finish migration of all remaining workload namespaces including gateways.`

We should probably include it here at the end and also at the beginning of the next "Create a default revision tag and re-label namespaces" chapture

Reply:
done